### ISSUE TITLE
How to search for specific objects of a specific class if I retrain the model with my dataset? 

### ISSUE THREAD

**Issue by satyambharti171** [AlexeyAB/darknet#6224]

**Comment by KacperPaszkowski** [comment]
If you trained model on own dataset it will only detect classes from your dataset.

**Comment by satyambharti171** [comment]
No I mean a specific object based on its features.

**Comment by KacperPaszkowski** [comment]
Any example? You train model for classifying those specific objects. If you have class 'dog' and you want to classify 'husky' and 'samoyed' you need to train your model with this classes.

**Comment by satyambharti171** [comment]
No, If I trained my model for chairs and I have a wooden red chair with cushions and have certain pattern drawn on it. I want to localize chair class then compare the features.
